### ISSUE TITLE
Unable to obtain the accessibility element with the specified attribute at mouse location

### ISSUE THREAD

**Issue by MattKleinsmith** [rxhanson/Rectangle#1129]
Hi, I can't snap any window in any direction. Any ideas? I tried reinstalling.

macOS version: 13.2.1 (22D68)
Rectangle version: 0.67 (73)
Logs if applicable (In Rectangle menu, hold option, "View Logging..."): 

![CleanShot 2023-04-06 at 23 44 49](https://user-images.githubusercontent.com/8968171/230542655-144c77c8-4230-4fa3-8adb-a883c821a536.gif)

<img width="494" alt="CleanShot 2023-04-06 at 23 51 03@2x" src="https://user-images.githubusercontent.com/8968171/230543210-83c6233d-33da-4082-8e67-22c68a90fc54.png">

**Comment by MattKleinsmith** [comment]
Reinstalled Raycast
